lib/pages/chat/chat.dart

```diff
@@ -1328,13 +1328,16 @@ class ChatController extends State<Chat>
     }
   }
 
+  bool isCopyAttachmentsAvailable(Event event) =>
+      !(PlatformInfos.isFireFoxBrowser && event.hasAttachment);
+
   List<PopupMenuItem> _popupMenuActionTile(
     BuildContext context,
     Event event,
   ) {
     final listAction = [
       ChatContextMenuActions.select,
-      ChatContextMenuActions.copyMessage,
+      if (isCopyAttachmentsAvailable(event)) ChatContextMenuActions.copyMessage,
```

Comment:
so we support copy attachment? what kind of attachment we support? pdf, docs?

Reply:
all type of files displayed in message bubble